back/src/domain/inclusionConnectedUsers/useCases/GetInclusionConnectedUser.unit.test.ts

```diff
@@ -144,4 +146,28 @@ describe("GetUserAgencyDashboardUrl", () => {
       erroredConventionsDashboardUrl: `http://stubErroredConventionDashboard/${agency1.id}_${agency2.id}_${agency4.id}`,
     });
   });
+  it("doesn't return errored convention dashboard url when when agency kind isn't PE", async () => {
```

Comment:
Typo dans le nom du test (when when). On peut être plus précis en disant :

`doesn't return errored convention dashboard url when user has no agency of kind PE`